pkg/pillar/diskmetrics/diskmetrics.go

```diff
@@ -56,6 +56,18 @@ func ResizeImg(log *base.LogObject, diskfile string, newsize uint64) error {
 	return nil
 }
 
+//CreateImg creates empty image file with defined format and size
+func CreateImg(log *base.LogObject, diskfile string, format string, size uint64) error {
+	output, err := base.Exec(log, "/usr/bin/qemu-img", "create", "-f", format, diskfile,
+		fmt.Sprintf("%d", size)).CombinedOutput()
```

Comment:
Strconv.Itoa or whatever it is called is nicer that a Sprintf like this.

Reply:
Moved to `strconv.FormatUint`